src/components/m-accordion-item/index.js

```diff
@@ -0,0 +1,59 @@
+import { BaseComponentGlobal } from '../_abstract/component-types';
+import CustomEvent from '../../js/custom-event';
```

Comment:
very good! i really appreciate that you checked in the utility for already implemented solutions! very nice!